yt/yt/tests/library/yt_queue_agent_test_base.py

```diff
@@ -91,6 +92,24 @@ class ObjectAlertHelper:
     def __init__(self, alerts):
         self.alerts = alerts
 
+    def check_matching(self, category, text=None, attributes=None):
```

Comment:
I would prefer it if we shared code with the helper below. Either put assert vs raise with some callback, or maybe just return False in these cases and outward assert will fail?